fixtures/ios_app_with_xcframeworks/Tests/xcframeworksTests.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+import XCTest
+
+@testable import xcframeworks
```

Comment:
typo: `@testable import App`

Reply:
wondering if I could include the static lib fixture in next PR as follow up work is needed to support static libs, which hasn't accounted in this PR in regards to `binary path`.